pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
```

Comment:
I would add the pytest config here as well and enable printing coverage to terminal either in the pyproject.toml or in the tox file